content/locales/fr.ts

```diff
@@ -41,18 +41,98 @@ export const fr = {
     title: "Informations personnelles",
     breadcrumb: "Profil | création",
     info: "Veuillez remplir ces informations pour créer votre profil.",
+    namePlaceholder: "Saisisser votre nom complet",
+    usernamePlaceholder: "Saisisser votre nom d'utilisateur",
+    profilePlaceholder: "Saisissez l’URL de votre profil",
+    websitePlaceholder: "Entrez l’URL de votre site Web",
+    picturePlaceholder: "Saisissez l’URL de votre image",
+    zoneInfoPlaceholder: "Saisissez les informations de votre zone",
```

Comment:
```suggestion
    zoneInfoPlaceholder: "Saisissez les informations de votre zone",
```
I don't see how not-power-users could know what the should enter 🤔 
And that's why we shouldn't use placeholders as we do and for exemple here, use `"e.g: Europe/Paris` instead of `"Saisissez les informations de votre zone"`

Reply:
> I don't see how not-power-users could know what the should enter 🤔

Yeah I know, I made a task earlier regarding having a call with Romain to check every strings :).